include/config.h

```diff
@@ -0,0 +1,28 @@
+#ifndef CONFIG_H
+#define CONFIG_H
+
+#include "logger.h"
+
+struct config_general {
```

Comment:
Do we really need that many structs? Can't we just have a single struct with two fields?

Reply:
No, it's more like a personal preference, to have one struct for each portal to create some kind of namespace separation.  Currently all options could be in one struct.